### PR TITLE
Fix lr initialization on reload

### PR DESCRIPTION
**Pull request by DanBmh**
This fixes a problem if someone uses the `force_initialize_learning_rate` flag. 
In the current implementation we reset the learning rate to the old one in this case and the LR-reduction we did before has no effect.

### CODE REVIEW

File: training/mozilla_voice_stt_training/train.py [hunk 1]
```diff
@@ -636,17 +636,23 @@ def __call__(self, progress, data, **kwargs):
                         break
 
                     # Reduce learning rate on plateau
-                    if (FLAGS.reduce_lr_on_plateau and
-                            epochs_without_improvement % FLAGS.plateau_epochs == 0 and epochs_without_improvement > 0):
-                        # If the learning rate was reduced and there is still no improvement
-                        # wait FLAGS.plateau_epochs before the learning rate is reduced again
+                    # If the learning rate was reduced and there is still no improvement
+                    # wait FLAGS.plateau_epochs before the learning rate is reduced again
+                    if (FLAGS.reduce_lr_on_plateau and epochs_without_improvement > 0
```
Comment (lissyx) [inline training/mozilla_voice_stt_training/train.py:641]: there is no change of logic here, please avoid changing ordering

Comment (DanBmh) [reply training/mozilla_voice_stt_training/train.py]: You're right, this should improve readability but it's better to split this to an extra pr.

**Comment by lissyx** [comment]
I don't know that code very well, I'm unsure to miss oddities.

**Review by reuben — APPROVED** [review]

**Comment by reuben** [comment]
Opened as https://github.com/mozilla/DeepSpeech/pull/3286 to run tests.

**Comment by reuben** [comment]
Merged in the other (rebased) PR! Thanks!
